utils/core-modules/contracts/modules/NftModule.sol

```diff
@@ -32,7 +32,7 @@ contract NftModule is INftModule, ERC721Enumerable, InitializableMixin {
         string memory tokenName,
         string memory tokenSymbol,
         string memory uri
-    ) public {
+    ) external onlyIfNotInitialized {
```

Comment:
why are we adding `onlyIfNotInitialized` here? Similar to TokenModule, should be reinitializable